misc/extension_api_validation/4.0-stable.expected

```diff
@@ -443,7 +443,6 @@ Validate extension JSON: Error: Field 'classes/PopupMenu/methods/add_shortcut/ar
 
 Added optional argument. Compatibility methods registered.
 
-
```

Comment:
Nitpick, but we actually keep two lines between blocks, it's just the one adding GH-80852 which missed it.